CHANGELOG.md

```diff
@@ -1,5 +1,26 @@
 # SAS Viya Monitoring for Kubernetes
 
+## Version 1.2.3 (20SEP22)
+
+* **Overall**
+  * [FEATURE] - The V4M container now includes OpenSSL so that you can run the deployment with TLS.
```

Comment:
- We had a preferred order for items but I don't recall it right now.  Let's use FEATURE, FIX, UPGRADE.  So, group the two FEATURES together and then show the FIX (NOT! "BUG"). ;-)
- How about we simplify things: [FEATURE] OpenSSL added to V4M container.
- (And, technically, that might be a FIX rather than a FEATURE, but...)